lib/y2_logs_helper.pm

```diff
@@ -2,7 +2,7 @@ package y2_logs_helper;
 use testapi;
 use strict;
 use warnings;
-use version_utils qw(is_sle is_caasp);
+use version_utils 'is_sle';
```

Comment:
I remember about some "agreement"  to keep `qw()` even for single entities . To make add/remove functions more smooth :) . But because agreement is in quotes ( and there is no way to do real one in reality of this codebase ) it is up to you to follow it or not ...